frontend/src/stores/userStore.ts

```diff
@@ -50,6 +51,28 @@ export const useUserStore = defineStore('users', {
 			}
 		},
 
+		async updateProfile(id: number, updateProfileForm: UpdateProfileForm) {
+			try {
+				if (updateProfileForm.new_password === '') {
+					delete updateProfileForm.new_password;
+					delete updateProfileForm.new_password_confirm;
+					delete updateProfileForm.password;
+				}
+				await patchRequest(`users/${id}`, updateProfileForm);
+				await this.refreshData();
+				await router.push('/settings');
```

Comment:
is this needed? As in, do we always want to redirect to the settings page after a profile update? (it also seems to work without this line)

Reply:
```ts
this.errors.length = 0
```
after this would help hide previous errors

Reply:
this.errors.length = 0 ----->  = great!
and maybe it is nice to redirect to another page, to have a sense of the change?
if we want to also show our own profile in the profiles list we could redirect to that?
what would you think?

Reply:
okay I'm redirecting to our own profile but we dont show our owns maybe something to discuss